zipkin_thrift_span.go

```
@@ -58,9 +58,10 @@ func BuildZipkinThrift(s *Span) *z.Span {
 	}
 	timestamp := utils.TimeToMicrosecondsSinceEpochInt64(span.startTime)
 	duration := span.duration.Nanoseconds() / int64(time.Microsecond)
+	ip, _ := utils.ParseIPToUint32(span.tracer.hostIP.To4().String())
```

Don't ignore the error. Perhaps we can add it as a tag?